### PR TITLE
First step for adding travis-ci

### PR DESCRIPTION
**Pull request by rayfordj**

### CODE REVIEW

**Comment by rayfordj** [comment]
_This system has less than 8 GB of total memory. Please have at least 8 GB of total ram free before running the installer._
- https://travis-ci.org/RHsyseng/sturdy-chainsaw/builds/257391222#L9023
- https://travis-ci.org/RHsyseng/sturdy-chainsaw/builds/257391222#L9030


---

https://github.com/theforeman/foreman-installer/blob/58ebfd143b0e2a8b604ae83af829bdc49a5e0f0e/.travis.yml#L24

File: Dockerfile [hunk 1]
```diff
@@ -6,6 +6,20 @@ ENV LANG=en_US.utf8 \
 
 EXPOSE 53 67 68 69 80 443 3000 3306 5910-5930 5432 8140 8443
 
+### Atomic/OpenShift Labels - https://github.com/projectatomic/ContainerApplicationGenericLabels
+LABEL name="acme/foreman-katello-systemd" \
```
Comment (tchughesiv) [inline Dockerfile]: "RHsyseng/foreman" to more closely match makefile?

File: Dockerfile [hunk 2]
```diff
@@ -6,6 +6,20 @@ ENV LANG=en_US.utf8 \
...
+      vendor="Acme Corp" \
+      version="0.1" \
+      release="1" \
+      summary="Acme Corp's Foreman/Katello app" \
+      description="Foreman/Katello app will do ....." \
+### Required labels above - recommended below
+      url="https://www.acme.io" \
+      run='docker run -tdi --name ${NAME} \
```
Comment (tchughesiv) [inline Dockerfile]: maybe remove run label due to complexity of this image?

**Comment by rayfordj** [comment]
_ERROR: No installation scenario was selected, the installer cannot continue._
- https://travis-ci.org/RHsyseng/sturdy-chainsaw/builds/257415820#L9072

File: Dockerfile (hunk 1 above)
Comment (rayfordj) [inline Dockerfile]: Should be resolved with https://github.com/RHsyseng/sturdy-chainsaw/pull/9/commits/d0ffb469dfa6a37ab73cb35bb42fa2c7de1870a3

File: Dockerfile (hunk 2 above)
Comment (rayfordj) [inline Dockerfile]: Should be resolved with https://github.com/RHsyseng/sturdy-chainsaw/pull/9/commits/d0ffb469dfa6a37ab73cb35bb42fa2c7de1870a3

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  - tmp=`mktemp`
+  - echo '{"insecure-registries":["172.30.0.0/16"]}' > ${tmp}
+  - sudo mv ${tmp} /etc/docker/daemon.json
+  - sudo mount --make-shared /
+  - sudo service docker restart
+  - make lint
+
+script:
+  - make test
+  - docker ps
```
Comment (tchughesiv) [inline .travis.yml]: `docker ps` needed?

Comment (tchughesiv) [inline .travis.yml]: if anything, `docker images`

Comment (rayfordj) [inline .travis.yml]: nope, not needed. It's  a holdover from copying & editing an existing file.

I'll flip it to be an `docker images`

**Pull request merged**